data-model.json

```diff
@@ -0,0 +1,149 @@
+{   
+    "name": "Name1 Name2 County",  
+    "update_time": "yyyy-mm-ddThh:mmz",
+    "source_url": "POINT TO LANDING PAGES, IN CASE ENDPOINTS CHANGE", 
+    "meta_from_source" : "STORE IMPORTANT NOTES FROM SOURCE HERE",
+    "meta_from_baypd": "STORE IMPORTANT NOTES ABOUT OUR METHODS HERE",
+    "series": {
+        "cases": [
+            { "date": "yyyy/mm/dd", "cases": -1, "cumul_cases": -1},
+            { "date": "yyyy/mm/dd", "cases": -1, "cumul_cases": -1 }
```

Comment:
So I totally missed this before, but `yyyy/mm/dd` seems like a super unusual format. Could we use ISO 8601 dates instead? (`yyyy-mm-dd` — just swap the `/` for `-`.)